x-pack/plugins/security_solution/server/lists_integration/endpoint/handlers/exceptions_pre_import_handler.ts

```diff
@@ -14,7 +15,9 @@ export const getExceptionsPreImportHandler = (): ValidatorCallback => {
   return async ({ data }) => {
     const hasEndpointArtifactListOrListItems = [...data.lists, ...data.items].some((item) => {
       if ('list_id' in item) {
-        return ALL_ENDPOINT_ARTIFACT_LIST_IDS.includes(item.list_id);
+        return (
```

Comment:
@approksiu @elastic/security-defend-workflows just want to confirm this is indeed the desired behavior. I thought I recalled that exception lists of type `endpoint` were never supposed to be able to be imported as it's considered an "internal" exception list maintained by us. 

This fix would mean that users cannot:

- Import an exception list of type `endpoint`
- Import rules with exception lists of type `endpoint` (pretty sure the rule import would succeed but the exception portion would fail)

Reply:
Pinging @elastic/security-defend-workflows any feedback here?

Reply:
Thanks for looking into this @yctercero. Indeed this change would disable the ability for importing exception list of type `endpoint`. While digging around I found that [this change](https://github.com/elastic/kibana/pull/154991/files#diff-20a7a18aed8b78775e8d2517fc3b2bc3e8f2b46739e13f49b58838a7a5f5721bR46) that was introduced a while ago is causing the bug that is attached to this PR. That change is gating which list types can be duplicated (including `endpoint` list type). I'm not sure if that was an intentional change. If it was, then it does make sense with the changes here. 

If not then we need to fix the original source of error, i.e. to exclude the `endpoint` list type from lists that can not be duplicated.

cc @dasansol92

Reply:
After discussing this with Ash, here are some questions which would be nice to answer in order to understand how to solve this issue:

1. When we import `endpoint list`, what is the expected behaviour of the "Overwrite the existing list" and "Create new list"?

<img width="1664" alt="Screenshot 2024-03-19 at 15 29 46" src="https://github.com/elastic/kibana/assets/2700761/aab281c5-1228-40ba-87d0-5f63a6e2bb79">

2. Are we allowed to override `Endpoint Security Exception List`?
3. If we allow to "Create new list" out of the endpoint list, right now we end-up with a new list which has a `list_id` not equal to `endpoint_list` but still has `type='endpoint'`.
4. Is there a possibility of newly created endpoint list from step 3 to collide in some way with the original endpoint list?
5. If we allow to import endpoint list, what operations are allowed on the newly created endpoint list (edit, duplicate etc.)? If duplication is allowed, why would we then disable duplication on original endpoint list?

<img width="1662" alt="Screenshot 2024-03-19 at 15 45 15" src="https://github.com/elastic/kibana/assets/2700761/f20478c5-bd51-42ec-85ea-a60ae345f8ed">

Reply:
Thanks for the questions, @e40pud ! I think most of them would be for product to determine. From what I remember way back during initial implementation was that the endpoint security exception list was an internally maintained list that users could add/remove exceptions from, but not delete or modify the parent list at all. If that is still the case, then I would assume that the behaviors for endpoint list would be:

- User cannot delete
- User cannot duplicate
- User can export
- User can import and select to override in the sense of "replace the existing items in this list with the items I'm importing" - nothing would be modified of the parent list 

This is going off of maybe outdated thinking though (back from initial implementation when there were a lot of restriction around the endpoint list). @caitlinbetz @approksiu can you confirm what the expected behaviors should be for the endpoint exception list?

Reply:
per conversations with @approksiu - I think we are okay to prevent users from selecting "Create new list" in this scenario. This aligns with existing functionality where we prevent users from duplicating this list. If possible we should make it clear why this option is unavailable for this list.

Reply:
Thank for the reply @caitlinbetz. Would this work?

<img width="1746" alt="Screenshot 2024-04-10 at 11 36 15" src="https://github.com/elastic/kibana/assets/2700761/d41bbefb-fa7b-48de-a822-8d4254f34546">
 
I disabled `Create new list` checkbox and added a tooltip describing the reason.

Reply:
thanks @e40pud ! Perhaps for now we can say - "We only allow one Exception List for Endpoint Security." to make it clear as to why they cannot create a new list.